Dockerfile

```diff
@@ -36,7 +36,7 @@ FROM ubi
 RUN microdnf update -y          && \
     microdnf install -y yum     && \
     yum install -y ca-certificates \
-    strongswan net-tools which  && \
+    net-tools which  && \
```

Comment:
There might be users of strongswan out there. It'd be bad that things stop working when they upgrade their rke2. I'd prefer to leave it

Reply:
Actually this is also not installed in the old image (See here #7) in the new UBI image it just does not fail silently anymore